Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -119,7 +119,7 @@ - (void)createContents
     [self.attachmentButton addTarget:self
                               action:@selector(attachmentButtonPressed)
                     forControlEvents:UIControlEventTouchUpInside];
-    UIImage *attachmentImage = [UIImage imageNamed:@"btnAttachments--blue"];
```

Comment:
Are we able to remove `btnAttachments--blue` now?

Reply:
oh yeah! Will do.